conda_smithy/update_cb3.py

```diff
@@ -364,9 +371,9 @@ def add_compiler(name, p_name):
 if __name__ == '__main__':
     import argparse
     parser = argparse.ArgumentParser()
-    parser.add_argument("recipe")
-    parser.add_argument("output")
-    parser.add_argument("config")
+    parser.add_argument("recipe", help="Path to recipe meta.yaml")
+    parser.add_argument("output", help="Path where updated recipe is stored")
+    parser.add_argument("config", help="Path to conda_build_config.yaml file")
```

Comment:
Is this optional? Can it default to the default conda build yaml for conda forge?

Reply:
Yes. This main method is for dev purposes only. Running `conda smithy update-cb3` from the root of the feedstock is enough.